tests/plugins/resizing.test.ts

```diff
@@ -383,6 +384,28 @@ describe("Model resizer", () => {
       expect(model.getters.getRowSize(sheet.id, 0)).toBe(multiLineHeight);
     });
 
+    test("wrapped text update the row size", () => {
+      setStyle(model, "C1", { fontSize: 10, wrapping: "wrap" });
+      resizeColumns(model, ["C"], 100);
+      setCellContent(model, "C1", "multiples wrapped lines");
+
+      const cell = model.getters.getCell({ sheetId: sheet.id, col: 2, row: 0 })!;
+      const expectedHeight = getDefaultCellHeight(cell, 100);
+      expect(expectedHeight).toBeGreaterThan(DEFAULT_CELL_HEIGHT);
+      expect(model.getters.getRowSize(sheet.id, 0)).toBe(expectedHeight);
+    });
+
+    test("updating column size updates row height with wrapped text", () => {
```

Comment:
you can add a test where the cell content is updated such that it's shorter and no longer wraps